app/src/main/java/com/x8bit/bitwarden/data/autofill/fido2/model/PublicKeyCredentialRequestOptions.kt

```diff
@@ -0,0 +1,15 @@
+package com.x8bit.bitwarden.data.autofill.fido2.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Models the request options for a passkey request.
+ */
+@Serializable
+data class PublicKeyCredentialRequestOptions(
```

Comment:
`challenge` is the only required property. All others are optional. 

Spec ref: https://www.w3.org/TR/webauthn-2/#dictionary-assertion-options

Reply:
Are you sure? It does make sense from viewing that document, but when I remove either of those other two properties, the string doesn't decode 🤔

Reply:
We don't need to remove them, just mark them nullable so no exceptions get thrown if they're not present in the JSON.